src/index.js

```diff
@@ -17,8 +17,20 @@ const templates = (() => {
 })();
 
 module.exports = (options) => {
-  const rollbar = new Rollbar(omit(options, ['template']));
+  Joi.assert(options, Joi.object().keys({
+    rollbar: Joi.object().keys({
+      accessToken: Joi.string().optional(),
+      environment: Joi.string().optional(),
+      enabled: Joi.boolean().optional(),
+      verbose: Joi.boolean().optional()
+    }),
```

Comment:
These are optional because of they are optional in the Rollbar object. Should `accessToken` and `environment` be required?